spec/views/spree/admin/orders/edit.html.haml_spec.rb

```diff
@@ -35,4 +35,15 @@ def current_ability
       expect(rendered).to have_content("Order Total $36.00")
     end
   end
+
+  describe "actions dropwdown" do
+    it "contains all the actions buttons" do
+      render
+
+      expect(rendered).to have_content("Resend Confirmation")
+      expect(rendered).to have_content("Send Invoice")
+      expect(rendered).to have_content("Print Invoices")
+      expect(rendered).to have_content("Cancel Order")
```

Comment:
yeah, I didnt have much luck trying to get this to work. let's skip it.

however you can keep this spec and just add:
expect(rendered).to have_content("links-dropdown")

also, I tried the code but the actions menu is not showing up, I think you will need to add 

:coffee
  angular.bootstrap(document.getElementById("links-dropdown"),['admin.dropdown'])

to
_orders_links partial

Reply:
I was not able to make it work how you suggested, so I created a new test, specific for the partial.